### PR TITLE
Fixed TuneIn (Long time fix)

### PR DESCRIPTION
**Pull request by DarkVillager**
Fixed TuneIn.
t4
t3
t1
t2

### CODE REVIEW

File: websites/T/TuneIn/presence.ts [hunk 1]
```diff
@@ -10,87 +10,73 @@ const presence = new Presence({
...
-            data.details = `${title.substring(0, 125)}...`;
+  if (playerCheck) {
+    const liveCheck = document.querySelector("#scrubberElapsed");
+    if (!liveCheck) return presence.setActivity();
+    if (liveCheck.textContent === "LIVE") {
+      const pauseCheck = document
+        .querySelector("#innerAppContent")
+        .querySelectorAll('[data-testid="player-status-stopped"]')[0]
+        ? true
+        : false;
```
Comment (ImRodry) [inline websites/T/TuneIn/presence.ts:30]: Please don’t do this

File: websites/T/TuneIn/presence.ts [hunk 2]
```diff
@@ -10,87 +10,73 @@ const presence = new Presence({
...
-  if (error) presence.clearActivity();
-  else {
-    const playerCheck = document.querySelector(
-      ".player__playerContainer___JEJ2U"
-    )
+    playerCheck = document
+      .querySelector("#innerAppContent")
+      .querySelectorAll('[data-testid="player"]')
       ? true
       : false;
```
Comment (ImRodry) [inline websites/T/TuneIn/presence.ts:21]: Please don’t

File: websites/T/TuneIn/presence.ts [hunk 3]
```diff
@@ -10,87 +10,73 @@ const presence = new Presence({
...
+      const audioTime = document.querySelector("#scrubberElapsed").textContent,
+        audioDuration = document.querySelector("#scrubberDuration").textContent,
+        timestamp1 = presence.timestampFromFormat(audioTime),
+        timestamp2 = presence.timestampFromFormat(audioDuration),
+        timestamps = presence.getTimestamps(timestamp1, timestamp2),
+        paused = document
+          .querySelector("#innerAppContent")
+          .querySelectorAll('[data-testid="player-status-paused"]')[0]
+          ? true
+          : false;
```
Comment (ImRodry) [inline websites/T/TuneIn/presence.ts:58]: Please don’t

File: websites/T/TuneIn/presence.ts [hunk 4]
```diff
@@ -10,87 +10,73 @@ const presence = new Presence({
...
+        ? (await strings).pause
+        : (await strings).live;
+    } else {
+      title = document.querySelector("#playerTitle").textContent;
+      author = document.querySelector("#playerSubtitle").textContent;
+      const audioTime = document.querySelector("#scrubberElapsed").textContent,
+        audioDuration = document.querySelector("#scrubberDuration").textContent,
+        timestamp1 = presence.timestampFromFormat(audioTime),
+        timestamp2 = presence.timestampFromFormat(audioDuration),
+        timestamps = presence.getTimestamps(timestamp1, timestamp2),
```
Comment (ImRodry) [inline websites/T/TuneIn/presence.ts:53]: Please don’t declare variables that are only used once

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: websites/T/TuneIn/presence.ts (hunk 1 above)
Comment (DarkVillager) [reply websites/T/TuneIn/presence.ts:30]: Okay? What to do then instead of this?

Comment (ImRodry) [reply websites/T/TuneIn/presence.ts:30]: Remove the ternary operator simply

**Comment by DarkVillager** [comment]
Currently dont have access to a pc will fix asap

**Comment by EncryptedDev** [comment]
Inactive - reopen later

**Pull request closed without merging**